### ISSUE TITLE
api-experimental does not correctly display actions having embedded spaces in the name

### ISSUE THREAD

**Issue by mdeuser** [openwhisk/openwhisk#1770]
1. Create an action having a space in the name - i.e. "Hello World"
2. Create an API using this action
  `wsk api-experimental create /bp /rp get "Hello World"`
3. List the APIs
  `wsk api-experimental list`
4. Observe the action name is truncated
```
ok: APIs
Action                            Verb             API Name  URL
/NAMESPACE/Hello                    get                 /bp  URL
```

**Comment by mdeuser** [comment]
@csantanapr - assign to me please

**Comment by mdeuser** [comment]
Fixed by PR #1772; closing issue
